### ISSUE TITLE
Incompatibility with "Chat Heads"

### ISSUE THREAD

**Issue by Keksuccino** [Keksuccino/FancyMenu#648]
The game will crash when trying to open action config screens or other parts of FancyMenu where suggestion lists for text input boxes are used.

I can't really fix this on my end, so I opened an issue in Chat Head's GitHub: https://github.com/dzwdz/chat_heads/issues/82

**Comment by Keksuccino** [comment]
```
at net.minecraft.client.gui.components.CommandSuggestions$SuggestionsList.handler$zjo000$chat_heads$chatheads$fixOutOfBoundChatHeads(CommandSuggestions.java:551) ~[minecraft-merged-project-@fabric-1.20.1-loom.mappings.1_20_1.layered+hash.2198-v2.jar:?]
	at net.minecraft.client.gui.components.CommandSuggestions$SuggestionsList.<init>(CommandSuggestions.java:339) ~[minecraft-merged-project-@fabric-1.20.1-loom.mappings.1_20_1.layered+hash.2198-v2.jar:?]
	at de.keksuccino.fancymenu.util.rendering.ui.widget.editbox.EditBoxSuggestions$EditBoxSuggestionsList.<init>(EditBoxSuggestions.java:366) ~[main/:?]
	at de.keksuccino.fancymenu.util.rendering.ui.widget.editbox.EditBoxSuggestions.showSuggestions(EditBoxSuggestions.java:191) ~[main/:?]
	at de.keksuccino.fancymenu.util.rendering.ui.widget.editbox.EditBoxSuggestions.updateCommandInfo(EditBoxSuggestions.java:162) ~[main/:?]
	at de.keksuccino.fancymenu.util.rendering.ui.widget.editbox.EditBoxSuggestions.createWithCustomSuggestions(EditBoxSuggestions.java:81) ~[main/:?]
	at de.keksuccino.fancymenu.customization.action.actions.other.EditMinecraftOptionAction$EditMinecraftOptionActionValueScreen.initCells(EditMinecraftOptionAction.java:130) ~[main/:?]
	at de.keksuccino.fancymenu.util.rendering.ui.screen.CellScreen.init(CellScreen.java:86) ~[main/:?]
```

**Comment by Keksuccino** [comment]
Fixed in Chat Heads v0.10.31 !
